.buildkite/scripts/common/util.sh

```diff
@@ -170,3 +170,34 @@ npm_install_global() {
 download_artifact() {
   retry 3 1 timeout 3m buildkite-agent artifact download "$@"
 }
+
+
+vault_get() {
+  path=$1
+  field=$2
+
+  fullPath="secret/ci/elastic-kibana/migrated/$path"
+  if [[ "$VAULT_ADDR" == *"secrets.elastic.co"* ]]; then
+    fullPath="secret/kibana-issues/dev/$path"
+  fi
+
+  if [[ -z "${2:-}" ]]; then
+    retry 5 5 vault read "$fullPath"
+  else
+    retry 5 5 vault read -field="$field" "$fullPath"
+  fi
+}
+
+vault_set() {
+  path=$1
+  shift
+  fields=("$@")
+
+  fullPath="secret/ci/elastic-kibana/migrated/$path"
```

Comment:
Do we need to write new fields under the `migrated` directory?

Reply:
I think it's not necessary - this was the original path used where Brian migrated the secrets. I can re-run and migrate to a path without these

Reply:
I migrated everything to the `secret/ci/elastic-kibana/` path, tested it here: https://buildkite.com/elastic/kibana-kme-test/builds/230#018c3bef-c769-4204-80da-328eeeae46bd - the secrets were successfully loaded and accessed on the new addresses too.